tests/_utils/utils.js

```diff
@@ -338,8 +338,6 @@ export function assertTableCellStyle( editor, tableCellStyle ) {
  *		] );
  *
  * The above call will check if table has second column selected (assuming no spans).
- *
```

Comment:
This note is important here. I think that it could be improved but there's must be info about this IMO.